website/docs/r/ecs_task_definition.html.markdown

```diff
@@ -65,6 +65,27 @@ contains only a small subset of the available parameters.
 ]
 ```
 
+### With AppMesh Proxy
+
+```hcl
+resource "aws_ecs_task_definition" "service" {
+  family                = "service"
+  container_definitions = "${file("task-definitions/service.json")}"
+
+  proxy_configuration {
+    type = "APPMESH"
+    container_name = "applicationContainerName"
+    properties = {
+      IgnoredUID = "1337"
```

Comment:
Nit: This should be alphabetized and formatted similar to `terraform fmt` where all the equals signs align for readability, e.g.

```hcl
AppPorts         = "8080"
EgressIgnoredIps = "..."
IgnoredUID       = "1337"
# ...
```